lib/fpm/package/deb.rb

```diff
@@ -18,6 +18,7 @@ class FPM::Package::Deb < FPM::Package
     :after_install      => "postinst",
     :before_remove      => "prerm",
     :after_remove       => "postrm",
+    :after_remove_purge => "postrm",
```

Comment:
I named it that way because `purge` is a special case of `remove`, which happens _after_ remove. "after remove: purge" (`purge` is even a part of the `postrm` script.) 

That said, I've got no problem renaming the CLI parameter and the variables. :-) I'm OK with `after_purge`, but `on_purge` would be more accurate IMHO.

Please let me known which variant you prefer, and I'll edit the PR.